mycli/main.py

```diff
@@ -818,7 +827,22 @@ def cli(database, user, host, port, socket, password, dbname,
                 confirm_destructive_query(stdin_text) is False):
             exit(0)
         try:
-            mycli.run_query(stdin_text, table_format=table)
```

Comment:
We now have two sets of code for running a query -- one that continues to use `run_query` and one that uses this version. It looks like the big difference is the newline handling. Is there any way we could continue using the `run_query` function, but also remove the extra newline? Maybe a helper function that looks at the settings to determine if we should output a newline or something?